src/components/ExternalLink/index.js

```diff
@@ -15,7 +15,9 @@ const ExternalLink = ({ to, noIcon, children, intl }) => (
     aria-label={intl.formatMessage(i18n.openInNewWindow)}
   >
     {children}
-    {!noIcon && <ExternalLinkSVG />}
+    {!noIcon && <i className="material-icons coa-ExternalLinkMaterial">
+      open_in_new
+    </i>}
```

Comment:
Here we add an italic tag for the material icon `open_in_new`, we also add the class  `coa-ExternalLinkMaterial `